src/application/services/useNote.ts

```diff
@@ -166,15 +167,24 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
    * @param parentId - Id of the parent note. If null, then it's a root note
    */
   async function save(content: NoteContent, parentId: NoteId | undefined): Promise<void> {
+    const { userEditorTools } = useAppState();
+
     if (note.value === null) {
       throw new Error('Note is not loaded yet');
     }
 
+    /**
+     * Format specified tools
+     */
+    const specifiedNoteTools = content.blocks.map((block) => {
```

Comment:
What means specified note tools?

Reply:
these are tools, that would be specified in the api request